src/eva/core/utils/operations.py

```diff
@@ -0,0 +1,26 @@
+"""Functional operations."""
+
+import re
+from typing import Iterable, List
+
+
+def numeric_sort(item: Iterable[str], /) -> List[str]:
```

Comment:
what does the `, /` do here in the function args?

Reply:
it does not allow you to call the function with key-words; for example you can only do `numeric_sort(list)` and not `numeric_sort(item=list)`. This is useful for numeric / processes functions, like for example `add` `devide` etc when you can not have a "nice" argument name or its protected (`_item`), so the person will not do `numeric_sort(_item=list)`